### PR TITLE
dotCMS/core#28172 Addition of Cancel Button to AI Image Block Editor Modal 

### LINKED ISSUE #28172: Addition of Cancel Button to AI Block Editor Modal
### Parent Issue

https://github.com/dotCMS/core/issues/27632

### User Story

AI Block Editor modal in our application lacks a direct, intuitive option for users to cancel their editing session.

### Acceptance Criteria

- The "Cancel" button should be positioned next to the "Regenarate" button, but visually distinct to avoid confusion.
- Upon clicking "Cancel," any changes made within the modal should be discarded.
- A confirmation prompt may be considered if the modal includes significant or complex edits, asking users if they're sure they want to discard changes.

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

Confirmation: https://www.figma.com/file/JF3P9JQVJHQOujhdpCRmJG/Block-Editor?type=design&node-id=2764-15650&mode=design&t=2H6qGxfBMxKbHSly-0 

Design from Melissa UX/UI https://www.figma.com/file/JF3P9JQVJHQOujhdpCRmJG/Block-Editor?type=design&node-id=2083-15349&mode=design&t=mwfb1uztfFYl922u-0

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Add cancel button to the Ai Image dialog. 
* Add confirmation to cancel. 
* add tooltip when copy the text 
* don't break words on form. 

### Screenshots

https://github.com/dotCMS/core/assets/31667212/33506ff4-5d77-4a98-826a-f99e76ffe885

### CODE REVIEW

File: core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts [hunk 1]
```diff
@@ -125,6 +128,10 @@ export class AiImagePromptFormComponent implements OnChanges, OnInit {
 
     copyToClipboard(): void {
         navigator.clipboard.writeText(this.aiProcessedPrompt);
+        this.tooltipText = this.dotMessageService.get('Copied');
+        setTimeout(() => {
+            this.tooltipText = null;
+        }, 1000);
```
Comment (zJaaal) [inline core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts:134]: You can do this by adding the `life`attribute where you use the tooltip
https://primeng.org/tooltip#api.tooltip.props.life

**Review by zJaaal — CHANGES_REQUESTED** [review]
see my comment

File: core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts (hunk 1 above)
Comment (oidacra) [reply core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts:134]: https://www.primefaces.org/primeng-v15-lts/tooltip take a look if works.

**Review by oidacra — APPROVED** [review]

**Review by zJaaal — APPROVED** [review]

File: core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts (hunk 1 above)
Comment (hmoreras) [reply core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts:134]: After the first click, I need to clean the text to don't show the tooltip again until the next click.

**Review by KevinDavilaDotCMS — APPROVED** [review]

**Pull request merged**